src/java_bytecode/java_bytecode_typecheck.h

```diff
@@ -43,7 +43,7 @@ class java_bytecode_typecheckt:public typecheckt
   {
   }
 
-  virtual ~java_bytecode_typecheckt() { }
+  virtual ~java_bytecode_typecheckt() = default;
```

Comment:
Let's keep the old version in this PR. We can change this consistently in the whole code base in separate PR.